glados/whisper_cpp_wrapper.py

```diff
@@ -855,7 +855,7 @@ def add_library_search_dirs(other_dirs):
 
 # End loader
 
-add_library_search_dirs([])
```

Comment:
I'd recommend checking in an empty `lib` folder in the root of the repo (just add a `.gitkeep` empty file inside), and copying the required files from the submodules there.

Does `./submodules/*` actually work from here? I'd expect `./` here to be `/path/to/project/root/glados/` whereas the access to the `submodules` folder presumes `./` is simply `/path/to/project/roo/`